src/runtime/protocols/hypervisor/hypervisor.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+option go_package = "github.com/kata-containers/kata-containers/src/runtime/protocols/hypervisor";
```

Comment:
This change doesn't exist in the original patch. It is usually a good practice to mention manual modifications in cherry picked patches along with a justification.

Why is this line needed in main and not in CCv0 ?

Reply:
> This change doesn't exist in the original patch. It is usually a good practice to mention manual modifications in cherry picked patches along with a justification.

Hey Greg - that's a fair point there have been quite a lots of modifications to the original commits that I've cherry-picked in order to add unit tests, fix up todos and respond to review requests, so I should probably update most of them to say based on the original commit rather than a cherry pick

> Why is this line needed in main and not in CCv0 ?

This was requested by Beraldo above to line up with the best practice that the other `src/libs/protocols/protos` files use.